src/shared/seccomp-util.c

```diff
@@ -213,6 +231,84 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "sigreturn\0"
                 "time\0"
         },
+        [SYSCALL_FILTER_SET_FILE_SYSTEM] = {
+                /* Basic file system access (reading file properties, opening/creating files and directories for
+                 * read+write) */
+                .name = "@file-system",
```

Comment:
I don't really see the point of the split betwen @file-system and @file-system-ops. (And I also think no dash would be better: @filesystem...). "Advanced" is in the eye of the beholder. I'd rather split this into @filesystem-query (access, chdir, various getattr, stat, inotify, mmap, readlink).

fcntl, close can be moved to @basic-io.

Can we filter on open(…, O_CREATE) and mmap(…, PROT_WRITE) and move those to the latter group?